conda_project/project.py

```diff
@@ -17,6 +17,8 @@
 from subprocess import SubprocessError
 from typing import List, Optional, Tuple, Union
 
+from conda_lock._vendor.conda.core.prefix_data import PrefixData
```

Comment:
One word of caution. In Conda Lock 1.2 it is `conda_lock._vendor...`, but earlier versions use `conda_lock.vendor...`

Reply:
I was wondering why my local version was broken compared to CI, and then got fixed when I did `conda update`  ... (see [6b3ea42](https://github.com/conda-incubator/conda-project/pull/39/commits/6b3ea42a066831722a449266151ee896df4c237c))

We should probably pin to >1.2 unless we want to do a conditional import ...

Reply:
Pinning to 1.2 sounds ok to me.